db/migrations/20180320162959_users.js

```diff
@@ -9,6 +9,7 @@ exports.up = async (knex) => {
     table.text('lastName').notNullable();
     table.text('email').notNullable();
     table.text('role').notNullable();
+    table.boolean('completed').defaultTo(false);
```

Comment:
Will not have an effect on the approval, but I would rename `completed` to `registerCompleted`, to give more context.